src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -146,7 +148,7 @@ public ChassisSpeeds getSpeeds() {
 
   /** stops the drivetrain's movement */
   public void stop() {
-    move(new Vector(0, 0), 0);
+    move(ORIGIN, 0);
```

Comment:
I recommend instead making this a static member of the Vector class. We call this the zero vector. May I recommend `Vector.Zero`?

Reply:
that's probably a good idea so I'll do it, though my only worry is that then any code from anywhere can effectively just modify the location of zero

Reply:
although, I do find that kind of interesting as a concept, so maybe it's fine

Reply:
Vector.Zero should be marked final. Not sure if you can return const/final equivalent from a method in Java?

Reply:
or are you asking if I have an immutable version of Vector, in which case I could, but currently don't

Reply:
Hm, @garrettwitowski he means that it'd be like marking an array final. Marking an array final doesn't prevent you from modifying its contents. Yes I think there should be an ImmutableVector, but that is for a future time. For now we should just avoid modifying it...